Bento/Views/CollectionViewContainerCell.swift

```diff
@@ -19,6 +20,10 @@ final class CollectionViewContainerCell: UICollectionViewCell {
         fatalError("init(coder:) has not been implemented")
     }
 
+    deinit {
+        bind(nil)
```

Comment:
Should we maybe have an unbind?

Reply:
`bind(nil)` is kinda the same? 🙈

Reply:
The thing is that I don't want to create an impression that you need to do balanced calls.

Reply:
It's just to me it seems that we are using side effects of passing `nil` to the method.